operator/v1/types_ingress.go

```diff
@@ -308,6 +316,32 @@ type EndpointPublishingStrategy struct {
 	NodePort *NodePortStrategy `json:"nodePort,omitempty"`
 }
 
+// RouteAdmissionPolicy is an admission policy for allowing new route claims.
+type RouteAdmissionPolicy struct {
+	// namespaceOwnership describes how host name claims across namespaces should
+	// be handled. The default is Strict.
+	//
+	// Value must be one of:
+	//
+	// - Strict: do not allow routes to claim the same host name across namespaces.
+	// - InterNamespaceAllowed: allow routes to claim different paths of the same host name across namespaces.
+	//
+	// The default is Strict.
+	NamespaceOwnership NamespaceOwnershipCheck `json:"namespaceOwnership"`
```

Comment:
Should this be optional?  We may later add additional fields to `RouteAdmissionPolicy` without requiring that every field be specified.

Reply:
hmm, changing it to

```go
// +optional
	NamespaceOwnership NamespaceOwnershipCheck `json:"namespaceOwnership,omitempty"`
```

and then running `make update` isn't making any changes to the generated schema, which surprised me... I'll have to figure that out

Reply:
Oh — the default is optional...

Reply:
Fixed